### PR TITLE
Quick Fix for useAuction

### PR DESCRIPTION
**Pull request by light-fury**

### CODE REVIEW

File: src/hooks/useAuctionBids.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { Auction } from 'src/interfaces/Auction'
  * @param auctionId
  */
 export function useAuctionBids(auctionId: string) {
```
Comment (adamazad) [inline src/hooks/useAuctionBids.ts:14]: `useAuctionBids` is going to be removed in a future PR.

Comment (light-fury) [reply src/hooks/useAuctionBids.ts:14]: Okay, then I will revert all these changes after you made a fix. Thank you @adamazad

Comment (light-fury) [reply src/hooks/useAuctionBids.ts:14]: Should I remove this file then? @adamazad

**Comment by light-fury** [comment]
Hi @adamazad 
Can you please check this again?

**Review by adamazad — APPROVED** [review]

**Pull request merged**